src/main/java/Entities/Gradable.java

```diff
@@ -0,0 +1,25 @@
+package Entities;
+
+public interface Gradable {
+    double weightage = 0;
```

Comment:
Java interfaces can't have instance variables - only static and final constants. So, these variables won't belong to instances of classes that implement Gradable.

These instance variables need to go in the Test and Assignment classes. Not ideal, but unfortunately the only way I can think to do this because Java classes also can't inherit from multiple classes